src/components/RepositoryCardForRender.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Descriptions, Image } from 'antd';
+import { GithubRepositoryItem } from '../modules/github/types';
+
+interface RepositoryCardForRenderProps {
+    item: GithubRepositoryItem;
+}
+
+export const RepositoryCardForRender = (props: RepositoryCardForRenderProps) => {
```

Comment:
Try destructure props as the following, it would be easier for you:
```js
export const RepositoryCardForRender: React.FC<RepositoryCardForRenderProps> = ({ item }) => {
```